libvcx/src/aries/handlers/issuance/holder/states/offer_received.rs

```diff
@@ -40,3 +33,20 @@ impl From<(OfferReceivedState, ProblemReport)> for FinishedHolderState {
         }
     }
 }
+
+impl OfferReceivedState {
+    pub fn new(offer: CredentialOffer) -> Self {
+        OfferReceivedState {
+            offer,
+        }
+    }
+
+    pub fn get_attributes(&self) -> VcxResult<String> {
+        let mut new_map: HashMap<String, String> = HashMap::new();
```

Comment:
You can use `let mut new_map = serde_json::map::Map::new();` and then return `Ok(Value::Object(new_map).to_string())` online `49` - so there wouldn't be any chance of serialization error on rust typing level. 
(The `insert` method on `serde_json::map::Map` accepts as value only `Value`, which is known to be serializable.)